Preliminary UI refactoring before #9

n1k0 opened the pull request:
This is preliminary refactoring of the views before starting tackling #9. 

![](http://i.imgur.com/9AK7a7e.png)

I removed the fixed sidebar thing, which was ugly and malfunctioning. I switched the many radio buttons to be just simple dropdowns, which I'm finding quite usable and gains us a lot of real estate.

Feedback? @magopian

magopian commented on ui/View.elm:
```diff
@@ -10,85 +10,77 @@ import Types exposing (..)
+    in
+        div [ class "form-group" ]
+            [ label [] [ text filterName ]
+            , select [ class "form-control", onInput onClickHandler, value checkedFilter ] <|
```
is that format expected? Having the `<|` on the end of a line?

n1k0 replied:
that's elm-format autoformatting

n1k0 commented:
I fixed a bug with the select widgets sometimes losing track of their selected value. I've also added a button to clear all filters at once, as well as rendering information about the number of matching builds and the list of filters applied:

![](http://i.imgur.com/aODUhYx.png)

magopian commented on ui/View.elm:
```diff
@@ -10,85 +10,133 @@ import Types exposing (..)
+
+
+filterSelector : List String -> String -> String -> (String -> Msg) -> Html Msg
+filterSelector filters filterName checkedFilter updateHandler =
```
Now that the filters are options in a select and not radiobuttons anymore, we might want to rename `checkedFilter` to `selectedFilter`?

n1k0 commented:
I'm landing this.

n1k0 commented:
I'm gonna address your latest comments directly on master, because I'm crazy.
